### PR TITLE
SAIC-398 Added check router ips are not overlapping

### PR DESCRIPTION
**Pull request by ntymtsiv**
Check gw ips of external router on src and dst are different.

### CODE REVIEW

File: devlab/tests/generate_load.py [hunk 1]
```diff
@@ -824,14 +830,6 @@ def wait_until_vms_all_deleted():
             except Exception as e:
                 print "Tenant %s failed to delete: %s" % (tenant['name'],
                                                           repr(e))
-        sgs = self.neutronclient.list_security_groups()['security_groups']
```
Comment (Settis) [inline devlab/tests/generate_load.py]: Why are you leave security groups while do `clean_objects()`?

Comment (ntymtsiv) [inline devlab/tests/generate_load.py]: This piece of code is duplicated. We still have cleaning for security_groups.

Comment (Settis) [inline devlab/tests/generate_load.py]: Got it.

**Comment by Settis** [comment]
+1

File: devlab/tests/functional_test.py [hunk 2]
```diff
@@ -14,11 +14,21 @@
...
 from generate_load import Prerequisites
 from filtering_utils import FilteringUtils
 
 
+def get_cf_root_folder():
+    return os.path.dirname(os.path.dirname(os.path.split(__file__)[0]))
+
+sys.path.append(get_cf_root_folder())
+import cfglib
+cfglib.init_config(os.path.join(get_cf_root_folder(), config.cloud_ferry_conf))
```
Comment (roman-verchikov) [inline devlab/tests/functional_test.py]: Let's better pass configuration.ini as argument to the tests if possible. Relying on config's position from the code is very optimistic.

**Comment by ntymtsiv** [comment]
Path to CloudFerry config is configurable, it specified in config.py.
Do you mean pass config like this: "nosetests --tc-file configuration.ini"?
It will affect all scripts for CI.

**Comment by roman-verchikov** [comment]
Ok, I see. We need to refactor this separately. +1 for now

**Pull request merged**